src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ title, user, body, comments }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+      <p>
+        {' Posted by '}
+        <UserInfo {...user} />
```

Comment:
user can be undefined so add a condition to show this component only when user is exist